app/src/main/java/com/kickstarter/ui/activities/CommentsActivity.kt

```diff
@@ -18,7 +20,24 @@ class CommentsActivity :
     BaseActivity<CommentsViewModel.ViewModel>(),
     CommentsAdapter.Delegate {
     private lateinit var binding: ActivityCommentsLayoutBinding
-    private val adapter = CommentsAdapter(this)
+    private val adapter = CommentsAdapter(
+        this,
+        object : DiffUtil.ItemCallback<Any>() {
```

Comment:
Awesome!

Reply:
Could we extract it to a public class re-usable in other recyclerview adapters ?

Reply:
done